server/views/pages/bookAVisit/dateAndTime.njk

```diff
@@ -54,6 +55,10 @@
     <div class="govuk-grid-column-full">
       {% include "partials/errorSummary.njk" %}
 
+      {% if validationAlert | length %}
+        {{ mojAlert(validationAlert[0]) }}
+      {% endif %}
+
```

Comment:
This is rendering full-width. Designs look to be two-thirds? (Applies to `overbooking.njk` too.) Worth double-checking what it should be.